packages/appium/docs/en/developing/build-plugins.md

```diff
@@ -368,7 +371,58 @@ clients won't have nice client-side functions designed to target these endpoints
 to create and release client-side plugins for each language you want to support (directions or
 examples can be found at the relevant client docs).
 
-An alternative to this way of doing things is to overload a command which all WebDriver clients
+#### Extending the BiDi Protocol
+
+You can also make new commands accessible via the WebDriver BiDi (WebSocket-based) protocol. BiDi
+commands come in two parts: a "module", which is basically a container or namespace, and
```

Comment:
According to what we have discussed with @christian-bromann BiDi modules must also have a vendor prefix if they are not standard. For example, we follow the below naming strategy:

`${vendorPrefix}:${domain}.${eventName}`

An example of such event would be the custom `contextUpdated` event in xcuitest driver: `appium:xcuitest.contextUpdated`